### DISCUSSION TITLE
Image format parameter

### DISCUSSION THREAD

**Discussion by tbrouard** [labs4capella/python4capella#138] [Q&A] [answered]
Hello,

I was wondering if the possibility of setting up the format of an image to export will added in a futur version (especially the SVG format which is provided by sirius for exporting Capella diagrams).
If i understand correctly the source code, only the "JPG" format is possible.
Thanks in advance for your answer.
Thomas Brouard

**Comment by ylussaud** [comment, marked as answer]
This feature is not planed for the moment. If you need this feature you can contact [Obeo](https://www.obeosoft.com/en/) for commercial support.
